scene/3d/area_3d.cpp

```diff
@@ -223,6 +223,11 @@ void Area3D::_body_inout(int p_status, const RID &p_body, ObjectID p_instance, i
 	bool body_in = p_status == PhysicsServer3D::AREA_BODY_ADDED;
 	ObjectID objid = p_instance;
 
+	// Exit early if instance is invalid.
+	if (objid.is_null()) {
```

Comment:
How come here same signals are not emitted? Can you add a comment as to why it's different from other cases? (eg. this callback is called in other place at time x).

Reply:
The reason is that the signal needs an object, these do not have objects assigned, see the code and how it assumes the object is valid

Reply:
See the code below to see that if the object is invalid the signal won't be emitted

Reply:
Edit, so it looks like it was added like that 9 years ago by reduz (seems to be a trend of things I spot in Physics lately). I would say this is not intentional. Feel free to fix and add the same `if (!node ||` check.

Reply:
Adding this

Reply:
Done, code now matches the 2D case and the other 3D case